.travis.yml

```diff
@@ -9,5 +9,5 @@ script:
   - make test
 branches:
     only:
-      - master
+      - mrehan/convert-lti-to-django-app
```

Comment:
Should the branch restriction just be removed?  We usually want to run tests on all PRs.